srai/loaders/base.py

```diff
@@ -0,0 +1,22 @@
+"""Base class for loaders."""
+
+import abc
+from typing import Any
+
+import geopandas as gpd
+
+
+class BaseLoader(abc.ABC):
+    """Base abstract class for loaders."""
+
+    @abc.abstractmethod
+    def download(self, gdf: gpd.GeoDataFrame, **kwargs: Any) -> gpd.GeoDataFrame:
```

Comment:
I'm not sure all our loaders will necessarily download data. Maybe change the name of the method?

Reply:
I was looking at out UML, but `load` seems to be a better idea